src/store/saga/userSaga.ts

```diff
@@ -164,6 +197,13 @@ export default function* rootSaga(): Generator<StrictEffect, void, any> {
     yield all([
         yield takeLatest(UserActionTypes.SIGNUP_REQUEST, registerUserSaga)
     ])
+    yield all([
+        yield takeLatest(UserActionTypes.OAUTH_ACCESS, oAuthAccessUserSaga)
+    ])
+    yield all([
+        yield takeLatest(UserActionTypes.OAUTH_REQUEST, oAuthRequestUserSaga)
+    ])
+
```

Comment:
```suggestion
    yield all([
        takeLatest(UserActionTypes.OAUTH_ACCESS, oAuthAccessUserSaga),
        takeLatest(UserActionTypes.OAUTH_REQUEST, oAuthRequestUserSaga)
    ]),
```
Остальные так же сюда можно